data/SpeedReaderConfig.json

```diff
@@ -1,4 +1,55 @@
 [
+    {
+        "domain": "css-tricks.com",
+        "url_rules": [
+            "! Initial support aimed for almanac, articles, license, and screencasts",
+            "@@/about/",
+            "@@/contact/",
+            "@@/authors/",
+            "@@/archives/",
+            "@@/advertising/",
+            "@@/books/",
+            "@@/guides/",
+            "@@/jobs/",
+            "@@/newsletters/",
+            "@@/product/",
+            "@@/tag/",
+            "/video-screencasts/*/^",
+            "/\/[a-z0-9-]+\//"
```

Comment:
would want to specify the domain with the rules as well (css-tricks.com), otherwise it will match for pretty much any alphanumeric URL and then try to look up the rule by domain and fall back. Same for exception rules

Reply:
Oh, I thought `url_rules` was only checked after `domain` matched. I'll fix this.